tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "strict": true,
     "noEmit": true,
     "allowSyntheticDefaultImports": true,
+    "resolveJsonModule": true,
     "types": ["webpack-env", "app/javascript/declarations"]
   },
   "exclude": [
```

Comment:
1. Does this minify?
2. Does it target the correct version of JS (and/or add polyfill for older browsers)

Reply:
1. Yes, as it is just like any other bit of JavaScript (as JSON _is_ JavaScript)
2. No polyfill needed, as JSON is just regular JavaScript.
3. Nope.

FYI The reason this has to be enabled explicitly is that they want to discourage people from reading big JSON files in their projects (which is usually better done via an XHR request). In our case, it's fine.

Reply:
I'll check.